parlai/tasks/coqa/build.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import parlai.core.build_data as build_data
+import os
+import json
+
+
+def make_parlai_format(outpath, dtype, data):
+    print('building parlai:' + dtype)
+    fout = open(os.path.join(outpath, dtype + '.txt'), 'w')
+    for each in data:
+        s = ""
+        for question, ans in zip(each['questions'], each['answers']):
+            s += "text:"
+            if question['turn_id'] == 1:
+                s += each['story'] + '\n'
```

Comment:
Ahh so I think this is why the test is giving you issues. You have the first turn of the conversation as not having a label. The first turn of the conversation should have the story + the first question.

Reply:
hmm? What I did is only add the story when it is the first question?

Reply:
`label:story/n question1`